charts/chart_versions.yaml

```diff
@@ -20,7 +20,7 @@ charts:
   - version: 2.11.100-build2023051511
     filename: /charts/rke2-metrics-server.yaml
     bootstrap: false
-  - version: v4.0.2-build2023081103
+  - version: v4.0.2-build2023081104
```

Comment:
Are there any new images that come with the changes to the chart, or does the DHCP daemonset use an existing image?

Reply:
no new images, it's using busybox

Reply:
I guess we should include busybox in the list of images for air-gap

Reply:
It looks like we for some reason specifically excluded the busybox image from the chart image validation scripts. If we're going to use busybox, we probably shouldn't exclude it from the list.

Reply:
I think I excluded busybox from the validation script because it's used in several charts but is never added in `build-images` so I assumed this was done on purpose.
Also several charts use the `latest` busybox instead of set version.

Reply:
busybox added to `build-images`